ansible/playbooks/backup-volumes.yml

```
@@ -0,0 +1,110 @@
+---
+- name: Docker Volume Backup
+  hosts: "{{ target_hosts | default('enabled_hosts') }}"
+  gather_facts: yes
+  vars:
+    backup_type: "{{ backup_method | default('tar') }}"  # tar, zfs
+    backup_location: "{{ backup_path | default('/opt/docker-backups') }}"
+    stack_filter: "{{ stack | default('') }}"
+    retention_days: "{{ retention | default(7) }}"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)